ToDoApi copy/src/main/java/com/theironyard/Controllers/UserController.java

```diff
@@ -0,0 +1,99 @@
+package com.theironyard.Controllers;
+
+import com.theironyard.Entities.ToDoList;
+import com.theironyard.Entities.User;
+import com.theironyard.Repositories.ToDoListRepository;
+import com.theironyard.Repositories.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import java.util.List;
+
+/**
+ * Created by darionmoore on 1/9/17.
+ */
+@Controller
```

Comment:
This is supposed to be a rest controller which means none of the controllers are working correctly.